### ISSUE TITLE
inject a module-deps json file produced by browserify to analyse

### ISSUE THREAD

**Issue by wadouk** [auchenberg/dependo#23]
Hi,
I try to use dependo to analyse my dependency graph.
I think I have understand that --deps of browserify build a json in module-deps format.
Is it currently the json format expected by dependo ? (seems don't)
I prefer to plug dependo on json output instead of the real code, because of browserify transformers applied on my source (babelify for es6 and jsx).

Thanks for any help.

**Comment by auchenberg** [comment]
Dependo can parse AMD, CommonJS and ES6 modules. I don't know if ``module-deps`` is a new format generated by Browserify, if it is, support has to be added.

**Comment by wadouk** [comment]
Thanks for your quick answer.
Dependo has a -f option that could be json. Do you have a sample of the expected format ?

**Comment by auchenberg** [comment]
Yes, you can see the details here https://github.com/auchenberg/dependo/pull/21
